doc/classes/EditorSceneImporter.xml

```diff
@@ -47,7 +47,8 @@
 			<argument index="0" name="path" type="String" />
 			<argument index="1" name="flags" type="int" />
 			<argument index="2" name="bake_fps" type="int" />
-			<description>
+      <argument index="3" name="compress_flags" type="int" />
+      <description>
```

Comment:
There's a style issue here, using spaces instead of tabs for indentation.

You can use `godot --doctool` to fix it by generating the template from the source code instead of manual edition.

Reply:
yea I realized that was the issue, just pushed a new update that hopefully will take care of it :) 

Thanks for pointing it out!!